src/test/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=password
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+
+spring.jpa.open-in-view=false
+spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
I can see that your test profile is identical to the default one

first of all you need to understand that Spring profiles support inheritance - read [this](https://medium.com/javarevisited/getting-started-with-spring-boot-profiles-1e00159f0542)

after reading the article above you'll understand that current file should be named as `application-test.properties`

however since all the properties are identical to the ones in default profile (`application.properties`), it seems that you don't need this file at all

Reply:
yeah, but it was mentioned, that we need such file for preventing the test devastating or smth like that, so it's supposed to be there